can_testbench.py

```diff
@@ -1,23 +1,18 @@
 import sys
-from dataclasses import dataclass, field
-from collections import deque
-from typing import List
+from os import path
+import configparser
+import dataclasses
+import collections
+import typing
+import enum
 import cantools
-from cantools.database.can.message import Message
-from cantools.database.can.signal import Signal
+from cantools.database import can as toolcan
 from cantools.database.namedsignalvalue import NamedSignalValue
-import can
-import os
+import can as pycan
```

Comment:
Changed imports to avoid importing classes instead of modules, makes it less confusing when there are two can libraries that implement some of the same classes.